ee/clickhouse/queries/experiments/funnel_experiment_result.py

```diff
@@ -320,39 +321,37 @@ def calculate_probability_of_winning_for_each(variants: List[Variant]) -> List[P
 
 
 def validate_event_variants(funnel_results, variants):
+    errors = {"no-events": True, "no-flag-info": True, "no-control-variant": True, "no-test-variant": True}
```

Comment:
small nit: Make the string error names an Enum, since all these values are accessed pretty often, prevents typo-bugs